app/src/main/res/values/id.xml

```diff
@@ -7,35 +7,35 @@
     <item name="emptyness" type="id" />
     <item name="counter" type="id" />
     <item name="Level_layout" type="id" />
-    <item name="Kafel_Field" type="id" />
-    <item name="kafel1" type="id" />
-    <item name="kafel2" type="id" />
-    <item name="kafel3" type="id" />
-    <item name="kafel4" type="id" />
-    <item name="kafel5" type="id" />
-    <item name="kafel6" type="id" />
-    <item name="kafel7" type="id" />
-    <item name="kafel8" type="id" />
-    <item name="kafel9" type="id" />
-    <item name="kafel10" type="id" />
-    <item name="kafel11" type="id" />
-    <item name="kafel12" type="id" />
-    <item name="kafel13" type="id" />
-    <item name="kafel14" type="id" />
-    <item name="kafel15" type="id" />
-    <item name="kafel16" type="id" />
-    <item name="kafel17" type="id" />
-    <item name="kafel18" type="id" />
-    <item name="kafel19" type="id" />
-    <item name="kafel20" type="id" />
-    <item name="kafel21" type="id" />
-    <item name="kafel22" type="id" />
-    <item name="kafel23" type="id" />
-    <item name="kafel24" type="id" />
-    <item name="kafel25" type="id" />
-    <item name="kafel26" type="id" />
-    <item name="kafel27" type="id" />
-    <item name="kafel28" type="id" />
-    <item name="kafel29" type="id" />
-    <item name="kafel30" type="id" />
+    <item name="tile_Field" type="id" />
```

Comment:
dont know what java convention says about this kind of names, but to stay similar to Level_layout this should be Tile_field